api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -169,6 +169,40 @@ public async Task<IActionResult> UploadFile([FromRoute] long id, [FromQuery] lon
         return new JsonResult(new ContentModel(content));
     }
 
+    /// <summary>
+    /// Attach an existing video/audio clip as the content of this snippet.
+    /// Only a single file can be linked to content, each new attachment will overwrite.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <param name="version"></param>
+    /// <param name="files"></param>
+    /// <returns></returns>
+    [HttpPut("{id}/attach")]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(ContentModel), (int)HttpStatusCode.Created)]
```

Comment:
Does not return a 201.  It returns a 200.

Reply:
Fixed.

Reply:
I noticed I use "HttpStatusCode.Ok" instead of "OK" before I pushed. I've addressed this.